matchms/filtering/__init__.py

```diff
@@ -121,8 +121,8 @@
 from .metadata_processing.require_precursor_below_mz import \
     require_precursor_below_mz
 from .metadata_processing.require_precursor_mz import require_precursor_mz
-from .metadata_processing.require_valid_annotation import \
-    require_valid_annotation
+from .metadata_processing.require_valid_annotation import (
+    repair_not_matching_annotation, require_valid_annotation)
```

Comment:
I thought those functions should not be in the same file --> see my suggetsions in  #513 (but that can be discussed there)

Reply:
Sorry, I am now just a bit confused about the order of the different PRs and branches...
This one should be solved by other PRs, right?